.storybook/preview.tsx

```diff
@@ -66,6 +66,8 @@ const preview: Preview = {
           publishableKey={publishableKey}
           userIdentifierKey={userIdentifierKey}
           endpoint={seamEndpoint}
+          disableCssInjection
+          disableFontInjection
```

Comment:
Cool. Need to update a few more places though: `README.md`, `examples/basic/App.tsx` (remove css import), `examples/basic/index.html` (remove font import) and disable here: https://github.com/seamapi/react/blob/main/test/fixtures/react.tsx#L18

Reply:
a3917de